EpsilonCCS/Src/TransmitSerial.c

```diff
@@ -2,7 +2,37 @@
 
 #include "stm32f4xx_hal_uart.h"
 
+/*
+* Flashing Red LED: The USART line is has errored sending this message, will re-try
+* Solid Red LED: The USART Port has timed out, check if Hermes is on
```

Comment:
Some other program may be listening in the future, it's more future proof to not says Hermes but like "other end of serial port is initialized" or something like that.